spec/rubocop/cop/betterment/boolean_query_parameter_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Betterment::BooleanQueryParameter, :config do
+  context 'when using parameters' do
+    it 'registers an offense when defaulting to a boolean value' do
+      expect_offense(<<~RUBY)
+        class UserController < ApplicationController
+          def taxable
+            params.fetch(:taxable, false)
```

Comment:
My one point of feedback would be to ask if we should have a model with a `:boolean` attribute that has `default: false`, instead of doing this in the controller.

Reply:
That may be a more appropriate solution to recommend. It would basically imply that as soon as you are dealing with params that have booleans, you should be using a stronger typed model. What do you think?